cli/internal/fs/path.go

```diff
@@ -47,14 +47,10 @@ func AbsoluteSystemPathFromUpstream(s string) turbopath.AbsoluteSystemPath {
 }
 
 // GetCwd returns the calculated working directory after traversing symlinks.
-func GetCwd() (turbopath.AbsoluteSystemPath, error) {
-	cwdRaw, err := os.Getwd()
-	if err != nil {
-		return "", fmt.Errorf("invalid working directory: %w", err)
-	}
+func GetCwd(cwdRaw string) (turbopath.AbsoluteSystemPath, error) {
```

Comment:
I think we should keep the `os.Getwd()` call in here, but gate it with a `if cwdRaw == ""`